packages/babel-types/src/definitions/experimental.js

```diff
@@ -61,23 +61,3 @@ defineType("ExportNamespaceSpecifier", {
     }
   }
 });
-
-defineType("RestProperty", {
```

Comment:
Ok we can do that, although the visitors are still gone since ast is changed, so would that do much?